src/event.js

```diff
@@ -610,8 +610,7 @@ jQuery.event = {
 			event,
 			{
 				type: type,
-				isSimulated: true,
-				originalEvent: {}
```

Comment:
Maybe I'm losing track of the discussion, but we do need to make at least one code change, since, as it stands, preventing default on a simple ```focusin``` event causes an exception. IIUC we can make one of three changes:
  0. Revert @markelog's commit - this would cause us to violate the principle that all events must be well-formed, but that's OK, because it's never caused a problem before. OTOH, if we go this route, we cannot update the docs page as @dmethvin suggested, because it won't be that that "the property may or may not be there", but that it may or may not be a well-formed event.
  1. ```e.originalEvent = new jQuery.Event();``` - IINM this does not require a docs update, but degrades event performance, as @gibson042 [pointed out](http://irc.jquery.org/%23jquery-dev/default_%23jquery-dev_20150512.log.html#t05:49:49).
  2. ```delete e.originalEvent;``` - This requires the docs update mentioned by @dmethvin.